Automattic/jetpack
# CRM: Add workflow row

Fixes https://github.com/Automattic/zero-bs-crm/issues/3208

## Proposed changes:


This PR adds a workflow row component.

### Other information:


## Jetpack product discussion


NtYgueI8oAgiWGHTbHpnjh-fi-1_29

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






1. Review the code.
2. Put `add_filter( 'jetpack_crm_feature_flag_automations', '__return_true' );` in an mu-plugin.
3. Visit `/wp-admin/admin.php?page=jpcrm-automations#/automations/test-workflow-row`. Verify that the visuals match the figma designs. Note that this PR only creates the row component. The table and table headers will be created in the next PR, so the current table is just a testing placeholder that can be ignored.
